llama-index-core/llama_index/core/workflow/handler.py

```diff
@@ -25,11 +25,12 @@ async def stream_events(self) -> AsyncGenerator[Event, None]:
 
         while True:
             ev = await self.ctx.streaming_queue.get()
-            if type(ev) is StopEvent:
-                break
 
             yield ev
 
+            if type(ev) is StopEvent:
```

Comment:
This change should yield the `StopEvent` and then break thereafter.

Reply:
Not yielding the stop event was done on purpose, imagining that would be useless most of the times. What was the use case for it?

Reply:
Yeah I think I agree that it would be useless most of the time to the developer who built the Workflow, presuming they know what events proceed a `StopEvent`.

I guess it just makes it cleaner? It was just surprising to me to not see the `StopEvent` come in the stream. 

Out of curiosity, would we be able to make a similar useless claims to streaming `StartEvent`? (I personally like seeing StartEvent streamed...)

Reply:
That's totally fair if we want to stream both! My thought initially was "since I'm sending start and stop events automatically under the hood as part of the workflow runtime, I don't want to clutter user output" (ironically streaming start event was a bug in this regard).
But if they're useful I have no objection in making them visible in the stream!

Reply:
Sgtm! Thank you.